engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js

```diff
@@ -21,6 +21,15 @@ angular.module('Bastion.hosts').factory('Host',
         resource.prototype.hasSubscription = function () {
             return angular.isDefined(this.subscription_facet_attributes) && angular.isDefined(this.subscription_facet_attributes.uuid);
         };
+
+        resource.prototype.isRpmEnabled = function() {
+            return !this.isDebEnabled();
```

Comment:
I don't like this but I don't see a better way of doing it right now. The advantage is that it preserves the existing behavior (show rpm packages tab everywhere) except for debs